fastlane/docs/Codesigning/CommonIssues.md

```diff
@@ -0,0 +1,38 @@
+# Commong code signing issues
+
+## Xcode errors
+
+> Code Sign error: No matching provisioning profiles found: No provisioning profiles with a valid signing identity (i.e. certificate and private key pair) matching the bundle identifier "X" were found.
+
+The provisioning profile for the given app identifier "X" is not available on the local machine. Make sure you have a valid provisioning profile for the correct distribution type (App Store, Development or Ad Hoc) on the Apple Developer Portal, download and install it, and select the profile in the Xcode settings (See [XcodeProject.md](XcodeProject.md))
```

Comment:
Are there any fastlane tools that the user should use here to download the provisioning profile? (`sigh download` or something)